src/Config/Config.ts

```diff
@@ -67,6 +69,8 @@ export default class Config implements IConfigParams {
 
     this.autostart = params.autostart !== undefined ? params.autostart : true;
     this.claiming = params.claiming || false;
+    this.endpoints = params.endpoints || [this.providerUrl];
```

Comment:
Can `endpoints` and `providerUrl` be merged into a single parameter gracefully?

Reply:
I think it's better to keep the `Config` parameters number as low as possible, to avoid complexity.

`endpoints[0]` seems like a good solution, if we can find a way to easily add an array of endpoints as parameters to the CLI prompt.  Maybe do some parsing on the CLI - if the user provides only a single string then it will only be a single element in the `endpoints` array (and also used as `providerUrl`).

Not sure how the user would provide an array of urls through the command line though. Maybe a config file?

It wouldn't break the DApp/CLI so long as we remember to update them together with the `timenode-core` version bump. :)

Reply:
:ok_hand: